docs/pages/versions/unversioned/guides/adhoc-builds.md

```diff
@@ -0,0 +1,128 @@
+
+## 0. Prerequisites
+
+You'll need an [Apple Developer Account](https://developer.apple.com/programs)
+
+## 1. Install Expo CLI
+
+Expo CLI is the tool for developing and building Expo apps. Run `npm install -g expo-cli` (or `yarn global add expo-cli`) to get it.
+
+If you haven't created an Expo account before, you'll be asked to create one when running the build command.
+
+**Windows users** must have WSL enabled. We recommend picking Ubuntu from the Windows Store. Be sure
+to Launch Ubuntu at least once. After that, use an Admin powershell to run:
+`Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Windows-Subsystem-Linux`
+
+## 2. Start the build
+
+Run `expo client:ios`.
+
+### 2a. Provide Apple Credentials
+
+You are given a choice of letting the `expo` client create the
+necessary credentials for you, while still having a chance to provide
+your own overrides. Your Apple ID and password is used locally and
+never saved on Expo's servers.
+
+```bash
+$ expo client:ios
+[16:44:37] Checking if current build exists...
+
+[16:44:37] No currently active or previous builds for this project.
+[16:44:37]
+Please enter your Apple Developer Program account credentials. These credentials are needed to manage certificates, keys and provisioning profiles in your Apple Developer account.
+
+The password is only used to authenticate with Apple and never stored.
+
+? What\'s your Apple ID? xxx@yyy.zzz
+? Password? [hidden]
+✔ Authenticated with Apple Developer Portal successfully!
+[16:44:46] You have 4 teams associated with your account
+? Which team would you like to use? 3) ABCDEFGHIJ "John Turtle" (Individual)
+
+? Select an iOS distribution certificate to use for code signing:
+❯ [Create a new certificate]
+  [Upload an existing certificate]
+
+? Select an authentication token signing key to use for push notifications: (Use arrow keys)
+❯ Used in apps: @john-turtle/expo-client-home-xxx (Key ID: XXX)
+  [Create a new key]
+  [Upload an existing key]
+```
+
+We ask you if you'd like us to handle your Distribution Certificate or
+use your own. If you have previously used `expo-cli` for building a standalone app
+for a different project, then we'll ask you if you'd like to reuse your existing
+Distribution Certificate. Similar to the Android keystore, if you don't know what
+a Distribution Certificate is, just let us handle it for you. If you do need
+to upload your own certificates, we recommend following [this excellent guide on making a P12 file](https://calvium.com/how-to-make-a-p12-file/).
+**Note:** this guide recommends leaving the P12's password blank, but a P12 password
+is required to upload your own certificate to Expo's service. Please enter a password
+when prompted. We'll ask you similar questions about your choice regarding
```

Comment:
```
We'll ask you similar questions about your choice regarding Push Notifications service key and provisioning profile.
```
This references the fact that we provide the option to handle  these resources (rather than prompting for password). I've made this clearer by saying `We'll also help you handle your push notifs + provisioning profile` rather than `We'll ask similar questions`